tests/functional/src/main/java/org/mule/functional/api/classloading/isolation/PluginUrlClassification.java

```diff
@@ -17,10 +17,8 @@
  *
  * @since 4.0
  */
-public class PluginUrlClassification {
+public class PluginUrlClassification extends ServiceUrlClassification {
```

Comment:
Is strange that plugin URL classification extends the service one as there is no relation between service/plugin. Can be renamed to ArtifactUrlClassification?